.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macOS-latest ]
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Prepare Ubuntu
+      run: |
+        sudo apt-get update
```

Comment:
Is this necessary?
I don't think it makes a big time difference, but I'm just curious why this is needed.

Reply:
No, this is not really needed. I removed it.

Reply:
Oh maybe it might be a problem, see https://github.com/sfstoolbox/sfs-python/pull/166/checks?check_run_id=1467403029
I think the test failed because it was not able to find the package as the index where to look for it (or for which version) was not up-to-date?

I have now reverted my commit to test again.